languages/elixir/exercises/concept/regular-expressions/.docs/instructions.md

```diff
@@ -0,0 +1,77 @@
+You have been tasked to write a service which ingests events. Each event has a date associated with it, but you notice that 3 different formats are being submitted to your service's endpoint:
+
+- `"01/01/1970"`
+- `"January 1, 1970"`
+- `"Thursday, January 1, 1970"`
+
+You would can see there are some similarities between each of them, and decide to write some composable regular expression patterns.
+
+## 1. Write patterns for each numeric component
```

Comment:
This is a relatively technical description. Maybe something like:

```suggestion
## 1. Allow extracting the day, month and year from a date
```

This comment also applies to the other tasks

Reply:
This pattern doesn't extract though, it would just match the day, month and year from a date.  Could I have `Match the day, month, and year from a date`?

Reply:
I just don't want the student to be mis-led that they should write the capture group at this step.

Reply:
> I just don't want the student to be misled and believe that they should write the capture group at this step.

I understand. 

> Match the day, month, and year from a date

I like that!

Reply:
```suggestion
## 1. Match the day, month, and year from a date
```